src/fields/u32/fp/wrapper.rs

```diff
@@ -58,6 +57,85 @@ impl Fp {
         fiat::fp_square(&mut result, &self.0);
         Self(result)
     }
+
+    fn inverse(&self) -> Option<Fp> {
+        if self == &Fp::zero() {
+            return None;
+        }
+
+        const LEN_PRIME: usize = 377;
+        const ITERATIONS: usize = (49 * LEN_PRIME + 57) / 17;
+
+        let mut a = fiat::FpNonMontgomeryDomainFieldElement([0; 12]);
+        fiat::fp_from_montgomery(&mut a, &self.0);
+        let mut d = 1;
+        let mut f: [u32; 13] = [0u32; 13];
+        fiat::fp_msat(&mut f);
+        let mut g = [0u32; 13];
```

Comment:
I think it might be clearer in these inversion methods to define a constant `N_LIMBS` so it's clear that e.g. 13 is `N_LIMBS + 1`, 12 is `N_LIMBS`, etc.

Reply:
added global constant `N_LIMBS`